core/services/vrf/extraargs/types.go

```diff
@@ -0,0 +1,34 @@
+package extraargs
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/smartcontractkit/chainlink/v2/core/utils"
+)
+
+const functionSignatureLength = 4
+const boolAbiType = `[{ "type": "bool" }]`
+
+var extraArgsV1Tag = crypto.Keccak256([]byte("VRF ExtraArgsV1"))[:4]
+
+func FromExtraArgs(extraArgs []byte) (nativePayment bool, err error) {
```

Comment:
Should this be `FromExtraArgsV1` since it only parses and returns V1 extra args?

Reply:
Yes will change this.